lib/Excel/Writer/XLSX/Workbook.pm

```diff
@@ -1092,10 +1093,17 @@ sub _store_workbook {
     );
 
     # Store the xlsx component files with the temp dir name removed.
+    my $createtime = timegm(@{ $self->{_doc_properties}{created} || $self->{_createtime} });
+
     for my $filename ( @xlsx_files ) {
         my $short_name = $filename;
         $short_name =~ s{^\Q$tempdir\E/?}{};
-        $zip->addFile( $filename, $short_name );
+        my $member = $zip->addFile( $filename, $short_name );
+
+        # Set zip member timestamp to match the workbook's timestamp, so that
+        # generated files are stable when the workbook's "created" property is
+        # manually set.
+        $member->setLastModFileDateTimeFromUnix( $createtime );
```

Comment:
Overall the commit looks good and the commit message is good, which puts me in a bit less of a bad mood. However, it still isn't clear to me why this change is either required (for a bug) or desired (for a feature request). Could you clarify?

Also, what does stable mean here?